tests-legacy/Integration/PrestaShopBundle/Controller/Admin/SurvivalTest.php

```diff
@@ -193,47 +169,7 @@ public function getDataProvider()
      */
     private function logIn()
     {
-        $loggedEmployeeData = new \stdClass();
```

Comment:
Although this was ugly, I belive this was need for test to correctly work? Was it replaced by something else? Be careful that we don't lose some test coverage because some might be skipped for some reasons

Reply:
From what I understood, it is useless as we use [a mocked security admin middleware](https://github.com/PrestaShop/PrestaShop/blob/develop/tests-legacy/PrestaShopBundle/Mock/AdapterSecurityAdminMock.php) during tests so it doesn't need a logged user to access to BO